README.md

```diff
@@ -67,10 +67,11 @@ The following keys are loaded from the keyring on start:
 
 - `peer-key` Ed25519 private key (required)
 - `encryption-key` AES-128, AES-192, or AES-256 key (optional)
+- `identity-key` Secp256k1 private key (optional). This key is used for node's identity.
 
 A secret to unlock the keyring is required on start and must be provided via the `DEFRA_KEYRING_SECRET` environment variable. If a `.env` file is available in the working directory, the secret can be stored there or via a file at a path defined by the `--secret-file` flag.
```

Comment:
Just like this paragraph it will be nice to explain here what a node identity is.

Reply:
will do